openmoltools/openeye.py

```diff
@@ -62,13 +62,18 @@ def get_charges(molecule, max_confs=800, strictStereo=True, normalize=True, keep
 
     charged_copy = generate_conformers(molecule, max_confs=max_confs, strictStereo=strictStereo)  # Generate up to max_confs conformers
 
-    status = oequacpac.OEAssignPartialCharges(charged_copy, oequacpac.OECharges_AM1BCCSym)  # AM1BCCSym recommended by Chris Bayly to KAB+JDC, Oct. 20 2014.
+    # 2017.2.1 OEToolkits new charging function
+    status = oequacpac.OEAssignCharges(charged_copy, oequacpac.OEAM1BCCCharges())
 
     if not status:
-        raise(RuntimeError("OEAssignPartialCharges returned error code %d" % status))
+        # OEAssignCharges returns False if failure.
+        raise(RuntimeError("OEAssignCharges failed."))
 
     #Determine conformations to return
     if keep_confs == None:
+        logger.warning(
+            "keep_confs was set to None. Returned molecules will not be charged.")
```

Comment:
I think in this case the returned molecule is still charged, but the coordinates are reset to those of the passed molecules, or did something changed with the new OE?

Do you think it'll be ok to send these messages to `logger.debug` instead of `warning`? Otherwise it will be very hard to silent them in our applications.